snap/snapcraft.yaml

```diff
@@ -128,9 +128,9 @@ parts:
   packages-deb:
     plugin: nil
     stage-packages:
-      - mysql-server-8.0=8.0.36-0ubuntu0.22.04.1
-      - mysql-router=8.0.36-0ubuntu0.22.04.1
-      - mysql-shell=8.0.36+dfsg-0ubuntu0.22.04.1~ppa4
+      - mysql-server-8.0=8.0.37-0ubuntu0.22.04.3
+      - mysql-router=8.0.37-0ubuntu0.22.04.3
+      - mysql-shell=8.0.37+dfsg-0ubuntu0.22.04.1~ppa3
```

Comment:
out of ignorance: what does the `ppa4 -> ppa3` means here?

Reply:
i am not sure to be honest, the ~ in a package name is used to indicate a version number or possibly a build. may be it indicates that `8.0.37` was built 3 times? this is a total guess though

https://launchpad.net/~data-platform/+archive/ubuntu/mysql-archive -> contains `8.0.36...~ppa4` where as I got the version number for 8.0.37 from https://launchpad.net/~data-platform/+archive/ubuntu/mysql-shell

Reply:
It is the amount of changes/try to build PPA. Every time you dput the package you need to bump version to produce the next package. for 8.0.37 we were a bit more lucky (try #3 passed all tests) comparing to 8.0.36 (we build it 4 times). :-D